EXAMPLES.md

```diff
@@ -126,9 +139,9 @@ In this example, if you called `product.title()` then an exception would be thro
 
 You can also make more complex requests using custom queries (see below).
 
-### Building a cart
+## Build a cart
```

Comment:
@sleroux I've changed this to a second level heading -- does it makes sense to have the headings for "Cart line items based on selected options", "Native web view checkout", and "Apple Pay checkout" as third-level headings _under_ this category, or should the checkout sections perhaps be treated differently? (separate from the "Build a cart" task)